react/features/invite/actions.js

```diff
@@ -40,11 +40,7 @@ export function updateDialInNumbers() {
         const mucURL = hosts && hosts.muc;
 
         if (!dialInConfCodeUrl || !dialInNumbersUrl || !mucURL) {
-            dispatch({
-                type: UPDATE_DIAL_IN_NUMBERS_FAILED,
-                error: 'URLs for fetching dial in numbers not properly defined'
-            });
-
+            // URLs for fetching dial in numbers not defined
```

Comment:
Default deployments are not using this option, so no need to print anything if this setting is missing. Otherwise, it will print error on every dialog open. I see codes and auth url as an extra feature that can be added and not there by default ...